source/_includes/_adminpanel/domains-ce.rst

```diff
@@ -249,12 +252,93 @@ CERTIFICATE`. In the dialog, you can choose to use:
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. card:: Let's Encrypt Requirements
+
+   Before attempting to ask for a Let's Encrypt certificate, make sure
+   that:
+
+   #. **Public Service Protocol** and **Public Service Host Name** are
+      correctly set in the |adminui|'s :menuselection:`Domain -->
+      General Settings`
+
+   #. There is a Virtual Host correctly configured for the domain you
+      want the certificate
+
+   #. **A**, **AAAA**, and **NXDOMAIN** record in the domain's DNS
+      configuration
+
+   #. The domain has a valid |fqdn| that can be resolved from anywhere
+      (i.e., the domain must be publicly accessible)
+
+   #. The domain (i.e., the Node which installs the Proxy) is
+      reachable from the Internet. In case the proxy is behind a
+      firewall or other routing devices and can not be reached
+      directly, add some forwarding rule on the firewall or device.
+
+To correctly issue a Let's Encrypt certificate for your |product|
+installation, you should carry out the following steps.
+
+The starting point is to generate the certificate using the |adminui|
+button, as shown in the :ref:`previous section <ap-vhost>`. besides
+the message on the bottom right corner, you will receive in a few
+minutes an e-mail stating the success or failure of the certificate's
+generation.
+
+.. hint:: You can follow the process by checking the log file
+   :file:`/var/log/carbonio/letsencrypt/letsencrypt.log`, using the
+   :command:`tail -f` command from the CLI.
+
+In case of failure, the e-mail will report the errors encountered that
+you need to fix before attempting again. Take into account that if you
+continuously ask for a certificate without success, you can be
+temporarily be prevented to ask again.
+
+The message *Successfully received certificate* appears in the e-mail
+when the issue is successful, together with other information,
+including the expiry date, followed by a second confirmatory e-mail.
+
+At this point you can deploy the certificate on your
+infrastructure. Log in to the CLI and issue, as the ``zextras user``, the
+commands
+
+.. code:: console
+
+   zextras$ /opt/zextras/libexec/zmproxyconfgen
+   zextras$ /opt/zextras/bin/zmproxyctl reload
+
+You can now setup the automatic renewal of the certificate by adding a
+line in the system's crontab :file:`/etc/crontab`. Run the following
+command
+
+.. hint:: Make a copy of the crontab before proceeding, so you can
+   recover it in case of troubles.
+
+   .. code:: console
+
+      # cp /etc/crontab /etc/crontab.original
+
+.. code:: console
+
+   zextras$ SLEEPTIME=$(awk 'BEGIN{srand(); print  \
+   int(rand()*(3600+1))}'); echo "0 0,12 * * * root sleep $SLEEPTIME \
+   && certbot renew -q" | sudo tee -a /etc/crontab > /dev/null
+
+This command will add a line similar to ``0 0,12 * * * root sleep 3018
+&& certbot renew -q`` to the crontab.
+
+or simply add the line yourself, making sure to use a random value
+after ``sleep``.
```

Comment:
uhmmmm, it seems complicated and risky to me...  at this state of developement I suggest only the idea to schedule the "certbot renew -q" command as zextras user but I don't know your policies on this...